src/_product-updates/2024-05-21-single-change.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Single Change
+tags: [New]
+image: /images/changelog/single-change.png
+---
+
+![single-change.png](/images/changelog/single-change.png)
+
+The changelog allows the consumers of your API documentation to track changes as your documentation evolves. We have recently added several new features to the changelog, such as the [changelog diff](https://docs.bump.sh/product-updates/2023/11/02/changelog-diff/) to track for example the differences between two changes.
```

Comment:
There are 2 "to the changelog" (one in this paragraph, one in the following one). Maybe we can rephrase it a bit?